bin/proxy.js

```diff
@@ -0,0 +1,75 @@
+require('babel-register');
+
+const http = require('http');
+
+const bunyan = require('bunyan');
+const config = require('config');
+const cookie = require('cookie');
+const httpProxy = require('http-proxy');
+
+const log = bunyan.createLogger({
+  name: 'proxy',
+  app: config.get('appName'),
+  serializers: bunyan.stdSerializers,
+});
+
+const proxy = httpProxy.createProxyServer();
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+log.info(`apiHost: ${apiHost}`);
+log.info(`frontendHost: ${frontendHost}`);
+
+const array = (value) => {
+  if (!value) {
+    return [];
+  } else if (Array.isArray(value)) {
+    return value;
+  }
+  return [value];
+};
+
+function unsecureCookie(req, res, proxyRes) {
+  const proxyCookies = array(proxyRes.headers['set-cookie']);
+  // eslint-disable-next-line no-param-reassign
+  proxyRes.headers['set-cookie'] = proxyCookies.map((rewrittenCookie) => {
+    if (!req.connection.encrypted) {
+      return rewrittenCookie.replace(/;\s*?(Secure)/i, '');
+    }
+    return rewrittenCookie;
+  });
+}
+
+function getHost(req) {
+  const useDesktop = req.headers.cookie && cookie.parse(req.headers.cookie).mamo === 'off';
```

Comment:
Hmm, I guess this means you can't simply type `http://localhost:3000/en-US/developers/` in the same browser that you're developing the mobile site on (i.e. in two different tabs). That's probably fine since toggling the cookie is easy enough.